app/scripts/modules/dcos/pipeline/stages/runJob/runJobStage.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+import _ from 'lodash';
+
+const angular = require('angular');
+
+import {DOCKER_IMAGE_AND_TAG_SELECTOR_COMPONENT} from 'docker/image/dockerImageAndTagSelector.component';
```

Comment:
Don't need to import this - and shouldn't import like this across top-level modules. If you need to import something from docker, it should be handled the same way as core, i.e. `from '@spinnaker/docker`.

Since this component isn't an export of the docker module, it's safe to assume at this point the selector will be there.